com.unity.renderstreaming/Runtime/Scripts/RenderStreaming.cs

```diff
@@ -45,6 +45,15 @@ public sealed class RenderStreaming : MonoBehaviour
         [InitializeOnLoadMethod]
         static void InitializeOnEditor()
         {
+            /// todo(kazuki):: This is workaround.
+            /// When kicking the Unity Editor with batchmode flag on command line, The Unity Editor crashes 
+            /// caused by not unloading WebRTC native plugin. By this workaround, Some static methods of this
+            /// package don't work correctly when batchmode. These static methods depend on WebRTC API,
+            /// therefore the package initialization must be completed just after launching Editor.
+            /// In the future, we will remove this workaround after improving the initialization of the
+            /// WebRTC package.
+            if (Application.isBatchMode)
+                return;
             RenderStreamingInternal.DomainUnload();
             RenderStreamingInternal.DomainLoad();
             EditorApplication.quitting += RenderStreamingInternal.DomainUnload;
```

Comment:
I think this change will break the ability to run unit tests from the commandline.

The following might be an option, but it would still make editor tests not work when running batchmode:

`#if UNITY_EDITOR
         /// todo(kazuki):: This is workaround.
         /// When kicking the Unity Editor with batchmode flag on command line, The Unity Editor crashes 
         /// caused by not unloading WebRTC native plugin. By this workaround, Some static methods of this
         /// package don't work correctly when batchmode. These static methods depend on WebRTC API,
         /// therefore the package initialization must be completed just after launching Editor.
         /// In the future, we will remove this workaround after improving the initialization of the
         /// WebRTC package.
        [InitializeOnLoadMethod]
        static void InitializeOnEditor()
        {
            if(Application.isBatchMode)
                 return;
            RenderStreamingInternal.DomainUnload();
            RenderStreamingInternal.DomainLoad();
            EditorApplication.quitting += RenderStreamingInternal.DomainUnload;
        }

        [InitializeOnEnterPlayMode]
        static void InitializeOnEditorBatchMode()
        {
            if(!Application.isBatchMode)
                return;
                
            RenderStreamingInternal.DomainUnload();
            RenderStreamingInternal.DomainLoad();
            EditorApplication.quitting += RenderStreamingInternal.DomainUnload;
        }
#else`

Reply:
I added the workaround for auto testing in this commit 😓 
https://github.com/Unity-Technologies/UnityRenderStreaming/pull/742/commits/cbaefc32bdd34b469c21bcf63f0066f96860d4ac